doc/api/crypto.md

```diff
@@ -1046,6 +1046,76 @@ encoding of `'utf8'` is enforced. If `data` is a [`Buffer`][], `TypedArray`, or
 
 This can be called many times with new data as it is streamed.
 
+## Class: KeyObject
+<!-- YAML
+added: REPLACEME
+-->
+
+Node.js uses an internal `KeyObject` class which should not be accessed
+directly. Instead, factory functions exist to create instances of this class
+in a secure manner, see [`crypto.createSecretKey`][],
+[`crypto.createPublicKey`][] and [`crypto.createPrivateKey`][]. A `KeyObject`
+can represent a symmetric or asymmetric key, and each kind of key exposes
+different functions.
+
+Most applications should consider using the new `KeyObject` API instead of
+passing keys as strings or `Buffer`s due to improved security features.
+
+### keyObject.getType()
+<!-- YAML
+added: REPLACEME
+-->
+* Returns: {string}
+
+Depending on the type of this `KeyObject`, this function either returns
+`'secret'` for symmetric keys, `'public'` for public (asymmetric) keys or
```

Comment:
Is there use for an HMAC key? Or is that a secret key? Symetric is usually used to describe ciphers.

Reply:
I think it should be `secret`. I will update the docs shortly.